src/codegen.cpp

```diff
@@ -357,6 +357,7 @@ static Function *jlboundserrorv_func;
 static Function *jlcheckassign_func;
 static Function *jldeclareconst_func;
 static Function *jlgetbindingorerror_func;
+static Function *jlboundp_func;
```

Comment:
wow that's old https://github.com/JuliaLang/julia/commit/691b8be811904f6370735f667cd31ed8a4f45d51 and used in a grand total of 3 places before this PR - maybe it should get a new name that says what it does?